Assets/LeapMotionModules/ElementRenderer/Scripts/SupportInfo.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+
+[Serializable]
+public struct SupportInfo {
```

Comment:
When I first saw this file in the root scripts folder it struck me as as "Support" as in "Help" rather than "Do I support this", and it stuck out. Maybe something like FeatureSupportInfo?